sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/src/Generated/Models/A2AProtectedDiskDetails.cs

```diff
@@ -122,7 +122,7 @@ internal A2AProtectedDiskDetails(Uri diskUri, ResourceIdentifier recoveryAzureSt
         /// <summary> The type of disk. </summary>
         public string DiskType { get; }
         /// <summary> A value indicating whether resync is required for this disk. </summary>
-        public bool? IsResyncRequired { get; }
+        public bool? ResyncRequired { get; }
```

Comment:
rename this property back to `IsResyncRequired` to avoid the breaking change

Reply:
Updated